tests/Feature/SiteToolingTest.php

```
@@ -301,6 +301,31 @@ public function test_failed_uninstall_records_failed_status(): void
+            fn (SocketEvent $event) => $event->data->type === 'isolated-user.tooling-updated',
+        );
+    }
+
+    public function test_uninstall_marks_failed_when_tool_missing_from_registry(): void
+    {
+        $this->iuser()->setToolingStatus('ghost-tool', SiteToolingState::STATUS_UNINSTALLING);
+
+        dispatch_sync(new UninstallSiteToolingJob($this->isolatedSite, 'ghost-tool'));
+
+        $this->assertSame('uninstall_failed', $this->iuser()->toolingStatus('ghost-tool'));
+    }
```

PHPMD (2.15.0)

[error] The method test_uninstall_marks_failed_when_tool_missing_from_registry is not named in camelCase. (undefined)

(CamelCaseMethodName)